spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -32,16 +32,16 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
-[[microprofile4.1-RC1]]
-=== MicroProfile 4.1-RC1 (Q32021)
-MicroProfile 4.1-RC1 is based on Jakarta EE 8. Although Jakarta EE 8 is API backwards compatible with Java EE 8, Jakarta EE replaces Java EE dependencies with Jakarta EE dependencies for all MicroProfile specifications.
+[[microprofile4.1-RC2]]
+=== MicroProfile 4.1-RC2 (Q32021)
```

Comment:
```suggestion
=== MicroProfile 4.1
```

Reply:
I would leave out the rough schedule as it does not add much value and it will be short lived anyway.